packages/server/src/tests/jestEnv.ts

```diff
@@ -3,11 +3,7 @@ import { tmpdir } from "os"
 
 env._set("SELF_HOSTED", "1")
```

Comment:
Would be nice if we made this file consistent with:

https://github.com/Budibase/budibase/blob/develop/packages/worker/src/tests/jestEnv.ts

And used `process.env`